near-sdk/src/lib.rs

```diff
@@ -70,3 +70,7 @@ pub use serde;
 
 #[doc(hidden)]
 pub use serde_json;
+
+#[doc(hidden)]
+#[cfg(feature = "abi")]
+pub use schemars;
```

Comment:
This was actually left out on purpose, as exposing external crates is a harmful practice (@austinabell correct me if I am wrong).

Ideally SDK should have something like `#[derive(NearAbi)]` that would derive `JsonSchema` and/or `BorshSchema` automatically, but I am yet to find a technical way to do this (tried once a couple of months ago).

Reply:
yeah, the re-exports aren't a good pattern. Ideally, we have some wrapper to obfuscate the fact that we're using schemars, since that might change in the future.

I don't hold this opinion very strongly, but just trying to avoid a maintenance issue

Reply:
> Ideally SDK should have something like #[derive(NearAbi)] that would derive JsonSchema and/or BorshSchema automatically, but I am yet to find a technical way to do this

#891